app-backend/db/src/main/scala/MVTLayerDao.scala

```diff
@@ -54,40 +79,114 @@ object MVTLayerDao {
       z: Int,
       x: Int,
       y: Int
-  ): Query0[Array[Byte]] = {
-    fr"""WITH mvtgeom AS
-      (
+  ): Query0[LabelTileGeometry] = {
+    fr"""
         SELECT
-          ST_AsMVTGeom(
-            join_table_join.geometry,
-            ST_TileEnvelope(${z},${x},${y})
-          ) AS geom,
-          join_table_join.annotation_task_id,
+          annotation_labels.geometry,
+          ST_TileEnvelope(${z}, ${x}, ${y}) as envelope,
+          annotation_labels.annotation_task_id,
           annotation_label_classes.id as label_class_id,
           annotation_label_classes.name,
           annotation_label_classes.color_hex_code
         FROM
-          (annotation_labels join (select id, status from tasks) tasks on annotation_labels.annotation_task_id = tasks.id
+          (annotation_labels join tasks on annotation_labels.annotation_task_id = tasks.id
```

Comment:
I like removing parens because some poor soul in the future (maybe me) will probably copy and paste them wrong and then get a cryptic sql error

7d70c71